src/Symfony/Component/Asset/UrlPackage.php

```diff
@@ -112,11 +112,11 @@ public function getBaseUrl($path)
      *
      * @param string $path
      *
-     * @return string The base URL for the given path
+     * @return int The base URL for the given path
      */
     protected function chooseBaseUrl($path)
     {
-        return fmod(hexdec(substr(hash('sha256', $path), 0, 10)), count($this->baseUrls));
+        return (int) fmod(hexdec(substr(hash('sha256', $path), 0, 10)), count($this->baseUrls));
```

Comment:
just wondering: why using `fmod` is better than using the modulo operator here (`%`)?